issuance/issuer.go

```diff
@@ -152,7 +152,18 @@ func LoadChain(certFiles []string) ([]*Certificate, error) {
 
 // IssuerConfig describes the constraints on and URLs used by a single issuer.
 type IssuerConfig struct {
-	UseForRSALeaves   bool
+	// Active determines if the issuer can be used to sign precertificates. All
+	// issuers, regardless of this field, can be used to sign final certificates
+	// (for which an issuance token is presented), OCSP responses, and CRLs.
+	Active bool
```

Comment:
I think [all usage](https://github.com/letsencrypt/boulder/blob/327f96d281dd75b5b0ebeb0745b83a791fe361db/issuance/issuer_test.go#L36-L39) of `UsafeForECDSALeaves` and `UseForRSALeaves` should be swapped out for `Active`.